### PR TITLE
Feat/407 update assigned message

### PR DESCRIPTION
**Pull request by starlitnightsky**
Updates #407 

- Test for tips messages
https://github.com/starlitnightsky/Real-Estate/issues/13#issuecomment-1604190580

### CODE REVIEW

**Comment by pavlovcik** [comment]
###### Tips

Tips should be an `<h6>`

File: src/handlers/comment/handlers/assign.ts [hunk 1]
```diff
@@ -89,7 +89,10 @@ export const assign = async (body: string) => {
   const deadline_msg = endDate.toUTCString();
 
   let wallet_msg, multiplier_msg, reason_msg, bounty_msg;
-
+  const tips_msg = `\nTips:\n
+  - Use /wallet 0x4FDE...BA18 if you want to update your registered payment wallet address ${payload.sender.login}
```
Comment (pavlovcik) [inline src/handlers/comment/handlers/assign.ts:93]:
```suggestion
  - Use `/wallet 0x4FDE...BA18` if you want to update your registered payment wallet address ${payload.sender.login}
```

File: src/handlers/comment/handlers/assign.ts [hunk 2]
```diff
@@ -89,7 +89,10 @@ export const assign = async (body: string) => {
   const deadline_msg = endDate.toUTCString();
 
   let wallet_msg, multiplier_msg, reason_msg, bounty_msg;
-
+  const tips_msg = `\nTips:\n
```
Comment (pavlovcik) [inline src/handlers/comment/handlers/assign.ts:92]:
```suggestion
  const tips_msg = `\n###### Tips:\n
```

Comment (pavlovcik) [reply src/handlers/comment/handlers/assign.ts:92]: I'm not sure if this syntax works.

**Comment by pavlovcik** [comment]
I implemented my changes and cleaned up the code.

**Comment by pavlovcik** [comment]
@Draeieg do I need to merge this in for you to QA or can you handle this locally? I'm actually not realizing your full process.

**Review by 0xcodercrane — APPROVED** [review]

**Pull request merged**